generator/metaline_builder.cpp

```diff
@@ -0,0 +1,217 @@
+#include "generator/metaline_builder.hpp"
+#include "generator/routing_helpers.hpp"
+
+#include "indexer/classificator.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/varint.hpp"
+#include "coding/write_to_sink.hpp"
+
+#include "base/checked_cast.hpp"
+#include "base/logging.hpp"
+#include "base/string_utils.hpp"
+
+#include "defines.hpp"
+
+#include <cstdint>
+
+namespace
+{
+using Ways = std::vector<std::int32_t>;
```

Comment:
Again, no.

Had to do my own digging. Correct answer: because `<cstdint>` includes `<stdint.h>`, which we don't. And in that file all the integer types in the global namespace are defined.